tools/scripts/linux.mk

```diff
@@ -379,6 +384,11 @@ ifeq (y,$(strip $(TINYIIOD)))
 	@$(MAKE) -C $(LIBRARIES)/libtinyiiod re
 LIBS += -L $(LIBRARIES)/libtinyiiod/build -ltinyiiod
 endif
+ifeq (y,$(strip $(MBEDTLS)))
+	@$(MAKE) -C $(LIBRARIES)/mbedtls clean
```

Comment:
Why do you want to do the clean every time?

Reply:
moved to clean